modules/sqlite_helpers.py

```diff
@@ -152,3 +154,18 @@ def get_number_of_entries(sqlite_file, search=None):
         cursor.close()
         db.close()
     return count
+
+def increment_used_column(sqlite_file, alias: str, count=1):
+    db = sqlite3.connect(sqlite_file)
+    cursor = db.cursor()
+    
+    try:
+        sql = "UPDATE urls SET used = used + ? WHERE alias = ?"
+        cursor.execute(sql, (count, alias))
```

Comment:
this could help for debugging
```suggestion
        logging.debug(f"incrementing the used column alias {alias} by {count}")
        cursor.execute(sql, (count, alias))
```